### ISSUE TITLE
Http POST

### ISSUE THREAD

**Issue by eddytjhin** [RainerRoss/WEBSRVUTL#3]
can we do POST Request in here?
Do you have any example for POST request?
Thank you

**Comment by RainerRoss** [comment]
yes you can do POST-Request because the WEBSRVUTL handles GET and POST

Try this SQL for POST-Request with Parameters

values SYSTOOLS.HTTPPOSTCLOB(
  'http://www.myhofi.com/myapp/websrv11.pgm',
  null
  'id=1'
);

and the GET-Request like this

values SYSTOOLS.HTTPGETCLOB(
  'http://www.myhofi.com/myapp/websrv11.pgm?id=1',
  null
);

Here is the example https://github.com/RainerRoss/WEBSRVUTL/blob/master/Examples/WEBSRV11.SQLRPGLE

You can get the Request-Type in your RPG-Program like this
LocMethod  = %str(getenv('REQUEST_METHOD':dsapierr));

**Comment by eddytjhin** [comment]
I realy appreciate for your response... thanks again, i will try again..
and i will let you know..
